tracpro/static/less/trac.less

```diff
@@ -203,4 +203,44 @@ footer {
 /* Override datepicker colors */
 .ui-datepicker select.ui-datepicker-month, .ui-datepicker select.ui-datepicker-year {
   color: #000000;
```

Comment:
I'm fixing the scrunching by hiding the words on small screen sizes. Fix coming.